WcaOnRails/config/application.rb

```diff
@@ -65,7 +64,7 @@ class Application < Rails::Application
     end
 
     # Setup available locales
-    I18n.available_locales = Locales::AVAILABLE.keys
```

Comment:
@gregorbg Looks like we don't get the access to the database here. Do you have any suggestions on how to fix this?

Reply:
Reason why I'm unhappy with `Locales::AVAILABLE` is just because it will be duplicate data. The same data will be there in UserGroups table as well.

In that case, can we generate `locales.rb` manually during build or through some cronjob to make it updated to the database? I really don't know how feasible it is, just exploring the ideas.

Reply:
Hm... I haven't put any deep thought into this, but in my opinion it doesn't hurt to have a hard-coded set of locales and have some database entries which follow these hard-coded locales.

It has the benefit of easy validation -- we can check whether there are database entries which are not in the list. We can also easily add new entries (support a new locale on the website) without having to add an orphaned translator-less group to the DB.

Reply:
Okay restored locales.rb and changed application.rb. In this PR, the migration is needed to add proper name to the group. When you get time, could you please check if any other changes also has to be reverted?